examples/simulation/Tutorial2/macros/create_digis.C

```diff
@@ -6,6 +6,7 @@
  *                  copied verbatim in the file "LICENSE"                       *
  ********************************************************************************/
 
+/*
```

Comment:
I used to not add our FairRoot headers in the macros for now, exactly for this reason (does not yet work).

Instead of commenting them out, what about something like this:
https://github.com/FairRootGroup/FairRoot/blob/6031156f11d95b3e4bd4470d705a76bcfce98773/examples/simulation/Tutorial1/macros/run_tutorial1_binary.C#L9

(Disclaimer: I don't know, which symbols are defined or not defined in the interpreter!)

Reply:
The commenting was for now to see if things work as expected. I will go with the preprocessor statement for now.

I could work around the first crash by moving some code block from the header to the source file but ran into another crash right after.